.env

```diff
@@ -80,4 +80,7 @@ GRAFANA_SERVICE_HOST=grafana
 JAEGER_SERVICE_PORT=16686
 JAEGER_SERVICE_HOST=jaeger
 
+# Sentry
+SENTRY_OTEL_DEMO_DSN=https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563
```

Comment:
I can also remove this again. But this would be how we can set DSN locally and then have it overriden by test-factory (assuming the override works with how I use it in `docker-compose.yml`).